src/processing/getCurrentRates/index.ts

```diff
@@ -56,7 +57,14 @@ export default async function getCurrentRates(
 
     try {
         if (response.status === 200) {
-            return response.data;
+            const result = response.data;
+            if ('RateLimit' in result && 'max_calls' in result.RateLimit) {
```

Comment:
Do you have an example of the response by chance?

I am curious if the 300ms sleep is enough time or if it should be longer or if we can figure it programmatically based on the response data and/or headers.

Reply:
Yes I thought of doing that later, to calculate the exact time to wait, it’s possible.
```
RateLimit	Object { calls_made: {…}, max_calls: {…} }
calls_made	Object { second: 4, minute: 325, hour: 325, … }
second	4
minute	325
hour	325
day	3745
month	13490
total_calls	13492
max_calls	Object { second: 20, minute: 300, hour: 3000, … }
second	20
minute	300
hour	3000
day	7500
month	50000
```

Reply:
Done